tron/bin/check_tron_jobs.py

```diff
@@ -293,7 +276,7 @@ def get_relevant_action(*, action_runs, last_state, actions_expected_runtime):
                     return action_run
                 if action_state == 'running':
                     stuck_action_run_candidate = action_run
-    return stuck_action_run_candidate or action_runs[-1]
+    return stuck_action_run_candidate
```

Comment:
Why take out action_runs[-1]?

I think another part of the code assumes that the relevant action is not None: https://github.com/Yelp/Tron/blob/ddb1524315ea99c895d774844434249c1e6fc76f/tron/bin/check_tron_jobs.py#L108

Reply:
I can't come out with any example in which we should return action_runs[-1] except non-schedule case. I think we should have enough confidence as long as unit tests passing.

Reply:
My interpretation of the code is similar to the one suggested by @chlgit - that `action_runs[-1]` exists purely to catch the case where `last_state=NOT_SCHEDULED` - however, it could also be the case that this clause is triggered when we pass `last_state=NO_RUNS_TO_CHECK` since I imagine that doesn't map to an actual `action_state`

I'll go ahead and leave this part in since I'm unfamiliar with this code and am choosing to be safe.